fhir-parent/pom.xml

```diff
@@ -28,6 +28,9 @@
         <fhir-persistence-jdbc.index>MINIMAL_JSON</fhir-persistence-jdbc.index>
         <fhir-server-test.index>MINIMAL_JSON</fhir-server-test.index>
         <java.version>1.8</java.version>
+        <fhir-examples.properties>4.0.1-SNAPSHOT</fhir-examples.properties>
```

Comment:
maybe `fhir-examples.version` instead of `fhir-examples.properties`

Reply:
done